metadeploy/api/serializers.py

```diff
@@ -502,17 +512,21 @@ def validate(self, data):
                 )
             )
 
-        user_has_valid_token = all(user.token)
-        if not user_has_valid_token:
-            raise serializers.ValidationError(
-                _("The connection to your org has been lost. Please log in again.")
-            )
+        # @@@ TODO: What to do with this in the case of a Scratch Org?
+        if user:
+            user_has_valid_token = all(user.token)
+            if not user_has_valid_token:
+                raise serializers.ValidationError(
+                    _("The connection to your org has been lost. Please log in again.")
+                )
 
-        data["org_name"] = user.org_name
-        data["org_type"] = user.org_type
-        data["full_org_type"] = user.full_org_type
-        data["organization_url"] = user.instance_url
-        data["org_id"] = user.org_id
+        # @@@ TODO: What to do with this in the case of a Scratch Org?
+        data["org_name"] = user.org_name if user else None
+        data["org_type"] = user.org_type if user else None
+        data["full_org_type"] = user.full_org_type if user else None
+        data["organization_url"] = user.instance_url if user else None
+        # @@@ TODO: Why is this needed?
+        # data["org_id"] = user.org_id
```

Comment:
@wlonk These are other places (in this case I think creation of an install Job?) where `user` is being assumed. I think we can wait to fix this in https://trello.com/c/vEBPQHK1?